demos/immersed_fem/immersed_fem.py.rst

```diff
@@ -0,0 +1,115 @@
+Creating Firedrake-compatible meshes in Gmsh
+============================================
+
+The purpose of this demo is to summarize the
+key structure of a ``gmsh.geo`` file that creates a
+Firedrake-compatible mesh. For more details about Gmsh, please
+refer to the `Gmsh documentation <http://gmsh.info/#Documentation>`_.
+
+The first thing we define are four corners of a rectangle.
+We specify the x,y, and z(=0) coordinates, as well as the target
+element size at these corners (which we set to 0.5). ::
```

Comment:
So that pylit does not think that these "code blocks" are python, use:

```
.. code-block:: none
  
    GeoCommands

```

Reply:
thanks